src/test/resources/bundles/userCardExamples3/template/usercard_task.handlebars

```diff
@@ -40,19 +40,13 @@
             <td style="width:4%">
                 AT :
             </td>
-            <td style="width:10%">
-                <div class="opfab-input">
+            <td style="width:15%">
+                <div class="opfab-input opfab-hour-input">
                     <label> HOUR </label>
-                    <input style="text-align:center" size="10" maxlength="2" type="text" id="hours"  value="{{card.data.hours}}">
-                </div>
-            </td>
-    
-            <td style="width:10%">
-                <div class="opfab-input">
-                    <label> MIN </label>
-                    <input style="text-align:center" size="10" maxlength="2" type="text" id="minutes" value="{{card.data.minutes}}">
+                    <input type="time" id="hours">
                 </div>
```

Comment:
- Put TIME as label

- Does not work in edit card mode , you shall user value="{{card.data.hours}}:{{card.data.minutes}} in the input to get back the value from the edited card